ngi_pipeline/conductor/classes.py

```diff
@@ -35,7 +35,7 @@ def get_engine(self):
         try:
             return get_engine_for_bp(self.project, self.config, self.config_file_path)
```

Comment:
Most likely, we never before came to this point, when ngi_pipeline was not able to identify the engine to run analysis. So this exception never occured.